lib/msf/core/payload/python.rb

```diff
@@ -14,10 +14,12 @@ module Msf::Payload::Python
   #
   def py_create_exec_stub(cmd)
```

Comment:
This needs to be updated to a class method (using `self.`) so it can be called externally. I'd drop the `py_` prefix since it's in the `Python` module anyways and you're already using it that way. As it's written now, I'm pretty confident this would throw an error.

```suggestion
  def self.create_exec_stub(cmd)
```